sdk/storage/azblob/internal/generated/zz_appendblob_client.go

```diff
@@ -1,6 +1,3 @@
-//go:build go1.18
-// +build go1.18
```

Comment:
The code generator update removed these build statements from all the generated files. Is this expected? @jhendrixMSFT

Reply:
Yes. The build constraints are no longer required.